kroxylicious-systemtests/src/test/java/io/kroxylicious/systemtests/AcceptanceST.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.systemtests;
+
+import java.io.IOException;
+import java.time.Duration;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInfo;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.kroxylicious.systemtests.installation.kroxylicious.Kroxy;
+import io.kroxylicious.systemtests.steps.KafkaSteps;
+import io.kroxylicious.systemtests.steps.KroxySteps;
+import io.kroxylicious.systemtests.templates.strimzi.KafkaTemplates;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/**
+ * The type Acceptance st.
+ */
+class AcceptanceST extends AbstractST {
+    private static final Logger LOGGER = LoggerFactory.getLogger(AcceptanceST.class);
+    private final String clusterName = "my-cluster";
+
+    /**
+     * Produce and consume message.
+     *
+     * @param testInfo the test info
+     * @throws IOException the io exception
+     */
+    @Test
+    void produceAndConsumeMessage(TestInfo testInfo) throws IOException {
+        String topicName = "my-topic";
+        String message = "Hello-world";
+        int numberOfMessages = 1;
+        String consumedMessage = message + " - " + (numberOfMessages - 1);
+
+        // start kroxy
+        LOGGER.info("Given Kroxy in {} namespace with {} replicas", Constants.KROXY_DEFAULT_NAMESPACE, 1);
+        kroxy = new Kroxy(Constants.KROXY_DEFAULT_NAMESPACE);
+        kroxy.deployPortPerBrokerPlain(testInfo, 1);
+
+        LOGGER.info("And KafkaTopic in {} namespace", Constants.KROXY_DEFAULT_NAMESPACE);
+        KafkaSteps.createTopic(testInfo, topicName, clusterName, 1, 1, 1);
+
+        LOGGER.info("When {} messages '{}' are sent to the topic '{}'", numberOfMessages, message, topicName);
+        KroxySteps.produceMessages(topicName, Constants.KROXY_BOOTSTRAP, message, numberOfMessages);
+
+        LOGGER.info("Then the {} messages are consumed", numberOfMessages);
+        String result = KroxySteps.consumeMessages(topicName, Constants.KROXY_BOOTSTRAP, numberOfMessages, Duration.ofMinutes(2).toMillis());
+        LOGGER.info("Received: " + result);
+        assertTrue(result.contains(consumedMessage), "'" + consumedMessage + "' message not consumed!");
```

Comment:
We should adopt `assertThat` for more expression assertions.

Reply:
done